tests/test_triton_server.py

```diff
@@ -100,23 +104,24 @@ def test_server_config(self):
 
             # Make sure parsed value is the one from dict, check type too
             test_value = CLI_TO_STRING_TEST_ARGS[arg]
-            self.assertEqual(
-                test_value,
-                type(test_value)(value),
-                msg=f"CLI string contained unknown value: {value}")
+            self.assertEqual(test_value,
+                             type(test_value)(value),
+                             msg=f"CLI string contained unknown value: {value}")
+
+    def test_create_server(self, gpus=None):
+        if gpus == None:
+            gpus = ["all"]  # Avoid strange default args behavior
```

Comment:
If the comment above is applied you can remote this function as well.

Reply:
Updated.